ui/src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+import {Sidebar} from '../../components/Sidebar/Sidebar';
+import {TopBar} from '../../components/TopBar';
+import styles from './styles.module.scss';
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+export const Layout: React.FC<LayoutProps> = props => {
+  const [isSidebarOpenOnMobile, setIsSidebarOpenOnMobile] = useState(false);
+  return (
+    <div className={styles.container}>
+      <Sidebar
+        isSidebarOpenOnMobile={isSidebarOpenOnMobile}
+        setIsSidebarOpenOnMobile={setIsSidebarOpenOnMobile}
+      />
```

Comment:
These child components don't need to know when they are visible or not. This should be handled in the parent components.